akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpResponseParser.scala

```diff
@@ -17,19 +19,20 @@ import ParserOutput._
  */
 private[http] class HttpResponseParser(_settings: ParserSettings, _headerParser: HttpHeaderParser)
   extends HttpMessageParser[ResponseOutput](_settings, _headerParser) {
+  import HttpResponseParser._
   import HttpMessageParser._
   import settings._
 
-  private[this] var requestMethodForCurrentResponse: Option[HttpMethod] = None
+  private[this] var responseForCurrentResponse: Option[ResponseContext] = None
```

Comment:
Sounds a bit strange :) Was this meant to be called `responseContextForCurrentResponse`?